browser/ui/views/split_view/split_view_location_bar.h

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_SPLIT_VIEW_SPLIT_VIEW_LOCATION_BAR_H_
+#define BRAVE_BROWSER_UI_VIEWS_SPLIT_VIEW_SPLIT_VIEW_LOCATION_BAR_H_
+
+#include <memory>
+#include <string>
+
+#include "base/scoped_observation.h"
+#include "chrome/browser/ui/toolbar/chrome_location_bar_model_delegate.h"
+#include "components/prefs/pref_member.h"
+#include "components/url_formatter/url_formatter.h"
+#include "content/public/browser/web_contents_observer.h"
+#include "ui/base/metadata/metadata_header_macros.h"
+#include "ui/views/view.h"
+#include "ui/views/view_observer.h"
+#include "ui/views/widget/widget_delegate.h"
+
+namespace views {
+class Label;
+class ImageView;
+}  // namespace views
+
+class PrefService;
+class LocationBarModel;
+
+// A simple version of the location bar for secondary web view.
+// This location bar only shows host of the current page.
+//  e.g. http://foo.bar.com/1234 => foo.bar.com
```

Comment:
is this still accurate? I think we are showing `http` now, right (but only http)

Reply:
Nice catch, Removed